pctasks/cli/dev_requirements.txt

```diff
@@ -0,0 +1 @@
+file:./pctasks/core#egg=pctasks.core
```

Comment:
A similar pattern is used in [azure-sdk-for-python](https://github.com/azure/azure-sdk-for-python/) to specify dependencies to packages in the monorepo.